src/Api/RequestAttributesExtractor.php

```diff
@@ -29,19 +31,17 @@
      *
      * @throws RuntimeException
      *
-     * @return array
+     * @return array [$resourceClass, $collectionOperation, $itemOperation, $format]
```

Comment:
A plain old example in the phpdoc to give an idea of what value you may have is good enough (besides what you added) IMO, `@example` or not